### ISSUE TITLE
Feature request - Support TP-Link Archer VR1200v

### ISSUE THREAD

**Issue by MatteoR23** [AlexandrErohin/home-assistant-tplink-router#80]
Hello,

Would it be possible to support also TP-Link Archer VR1200v?

Thank you very much.
Best regards,
Matteo

**Comment by AlexandrErohin** [comment]
Have you tried? Archer VR2100v v1 is supported

**Comment by MatteoR23** [comment]
Yes, I tried it, but with no success, that's why I opened the feature request. 

BTW, you mentioned: 
```
 Archer VR2100v
```
is different than mine:
```
Archer VR1200v.
```

**Comment by AlexandrErohin** [comment]
As I dont have this extender - Would be great if you could add support for your router by creating PR with new class that extends https://github.com/AlexandrErohin/TP-Link-Archer-C6U/blob/main/tplinkrouterc6u/client.py#L19

**Comment by AlexandrErohin** [comment]
Hi. Could you try the latest integration version?

**Comment by MatteoR23** [comment]
Hi @AlexandrErohin, sorry I forgot about this issue opened 5 months ago 😄 It works, I close it
